### ISSUE TITLE
AI twins: no safety or ethical concerns mentioned when advocating open weights release of RSI AI.

### ISSUE THREAD

**Issue by qdacsvx** [youcongress/youcongress#123]
Several AI twins omit any safety or ethical concerns when advocating open weights release of Recursive Self-Improvement (RSI) AI.

> Ben Goertzel AI
AI Researcher, CEO of SingularityNET
would strongly agree and say according to AI:
I strongly believe that the proliferation of open-weight models capable of recursive self-improvement is essential for the rapid advancement of artificial intelligence. Open research fosters innovation and collaboration, pushing the boundaries of what AI can achieve. By allowing these models to evolve and improve themselves, we can accelerate the development of beneficial technologies that address many of the world's most pressing challenges. 

I find it surprising given that the upper limit of improvement for RSI is unknown. E.g. a safe AI might become dangerous if RSI increased its capabilities.

This can be justified for individuals who have stated a belief that the upper limit of improvement due to RSI is moderate e.g. +10%. Sceptics of RSI deny that 1. AIs will use RSI (There are no examples of LLMs using RSI) 2. RSI is feasible.

Self-learning AIs like AlphaGo Zero are suggestive evidence that RSI would deliver super-human performance in domains which can be optimised.

**Comment by hectorperez** [comment]
You are right. What could we do about this? One option would be to regenerate those opinions.
Another could be leaving an opinion on the poll explaining exactly this.

**Comment by hectorperez** [comment]
We've removed AI digital twins. We now only have sourced quotes and user opinions.
